### ISSUE TITLE
Use of non-standard name for id-column in UserModel

### ISSUE THREAD

**Issue by rrrealman** [Flask-Middleware/flask-security#312]
Hello everyone!
I encountered an old flask-security issue.
Assuming this code:
```
class UserModel(db.Model, UserMixin):

    __tablename__ = 'users'

    user_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
    first_name = db.Column(db.String(length=255))
    last_name = db.Column(db.String(length=255))
    email = db.Column(db.String(length=255), unique=True)
    password = db.Column(db.String(length=255))
    active = db.Column(db.Boolean())
    confirmed_at = db.Column(db.DateTime())
    roles = db.relationship(
        'RoleModel', secondary=roles_users,
        backref=db.backref('users', lazy='dynamic'))

    def __str__(self):
        return self.email

    def get_id(self):
        return self.user_id
```
Here I'm using `id` column name different from just `id` (namely, `user_id`) because I don't like to mess up column name with an embedded function `id()`. To make it work I've implemented `get_id` getter, as it is recommended in flask-login reference and have injected custom `user_loader` into flask-login:
```
def install_user_loader(app):
    '''In order to handle non-standard id column name'''
    with app.app_context():
        security = app.extensions['security']
        @app.login_manager.user_loader
        def user_loader(user_id):
            return security.datastore.find_user(user_id=user_id)
```
 After all that has been done I got multiple errors while performing login/logout or admin `AttributeError: 'UserModel' object has no attribute 'id'`. As I see, regular way to get user id is invoking `get_id` method, that incapsulates all needed logic to decide which attribute to use. 
So, I've made simple patch for flask-security. Please, have a look at it.

**Comment by jwag956** [comment]
SO I need to look at this more - but thinking out loud here - possibly a pretty easy answer is to have a configuration variable called 'SECURITY_USERMODEL_ID_COLUMN' or something like that - that by default will be set to fs_uniquifier if is exists else 'id' - but could be set by application to be anything. get_id or other places that need to search by 'id' would use that as the column name rather than anything hardoded....

**Comment by jwag956** [comment]
Fixed via #339
